tests/integration_tests/fail_over_ddl_mix/run.sh

```diff
@@ -216,7 +216,7 @@ main() {
 	# kill -9 $NORMAL_TABLE_DDL_PID $PARTITION_TABLE_DDL_PID $DML_PID_1 $DML_PID_2 $DML_PID_3 $DML_PID_4 $DML_PID_5 $DML_PID_6 $DML_PID_7 $DML_PID_8 $DML_PID_9 $DML_PID_10
 	kill -9 $NORMAL_TABLE_DDL_PID $DML_PID_1 $DML_PID_2 $DML_PID_3 $DML_PID_4 $DML_PID_5
 
-	sleep 30
+	sleep 60
```

Comment:
not the root cause, we need to find the root cause or a better way to avoid it failed